.travis.yml

```diff
@@ -38,6 +38,7 @@ branches:
   - "/^feature/#1009_History_of_pointvalue_No_annotation.*$/"
   - "/^feature_s/#1094_.*$/"
   - "/^feature_s/#1107_.*$/"
+  - "/^feature_s/#1112_.*$/"
```

Comment:
"/^feature/#1112_.*$/"

Reply:
Thank you

Reply:
1. Create data point virtual data source - enabled change random
![Screenshot from 2020-02-12 13-56-38](https://user-images.githubusercontent.com/1280575/74337820-55a26180-4da1-11ea-8fc4-434845838406.png)
there is no indication on the user in the annotation - (this is not a big problem because the system has changed its value)
2. Create data point virtual data source - enabled change random and settable
 - Change handly value and random (system).
 - Disable data point.
 - Enable data point and don't have annotation users
![Screenshot from 2020-02-12 14-07-41](https://user-images.githubusercontent.com/1280575/74338061-c9dd0500-4da1-11ea-90f3-94d7d386f960.png)

And it doesn't work right here.

Reply:
I imported json data and 

![after_import_grzesiek_json](https://user-images.githubusercontent.com/3029545/74342710-352ad500-4daa-11ea-9fa8-91a4f4531026.png)

Reply:
In the imported json conf we need fix name of one existing point because is null. Nevermind.
Here is new one test on fresh machine and fresh database.

![new_scada_on_new_database](https://user-images.githubusercontent.com/3029545/74351602-a02ed880-4db7-11ea-9b5a-fca6eadf08fb.png)


![inside_point_details](https://user-images.githubusercontent.com/3029545/74351817-ed12af00-4db7-11ea-8757-319e3a7b0786.png)

Reply:
After importing the data this is a different scenario than I am changing the data and then I see that there is no information that I have changed the data.

Reply:
Different scenario. You showed me another scenario earlier, so that is hiding of knowledge. It can't be like that.If you changing data in , ways , without Scada, please be aware that you are not helping to solve the problem.Please share all ways , which you do.

Reply:
Create data point virtual data source - enabled change random and settable
Change handly value and random (system).
Disable data point.
Enable data point and don't have annotation users

Reply:
I have done same steps and code which I was fixing is , in this case, not used, so that's why that case in as other case. Change value by user in standard case and change value by rest works.

Reply:
I have a little different behavior on docker 1112 now.
For the scenario that's described above.
The change history shows that the user who has passed the data has been deleted, even though he was not deleted.
![Screenshot from 2020-03-02 12-17-36](https://user-images.githubusercontent.com/1280575/75673004-3c931f00-5c82-11ea-99a1-d8315da90998.png)
![Screenshot from 2020-03-02 12-18-06](https://user-images.githubusercontent.com/1280575/75673049-57659380-5c82-11ea-9dd7-d290b0cbe6b5.png)
![Screenshot from 2020-03-02 12-18-28](https://user-images.githubusercontent.com/1280575/75673127-7b28d980-5c82-11ea-8029-c583fb86e142.png)

Reply:
I still have the same problem
![Screenshot from 2020-03-09 13-55-22](https://user-images.githubusercontent.com/1280575/76214937-fe5fa780-620d-11ea-9b63-58337034d10d.png)
![Screenshot from 2020-03-09 13-55-51](https://user-images.githubusercontent.com/1280575/76214947-015a9800-620e-11ea-86b9-a2b9d48c9942.png)